src/mame/drivers/contra.cpp

```diff
@@ -59,10 +59,57 @@ void contra_state::contra_coin_counter_w(uint8_t data)
 		machine().bookkeeping().coin_counter_w(1, (data & 0x02) >> 1);
 }
 
+uint8_t contra_state::contra_K007452_r(offs_t offset)
+{
+	switch (offset)
+	{
+		case 0:
+			return m_multiply_result & 0xFF;
+			break;
+		case 1:
+			return (m_multiply_result >> 8) & 0xFF;
+			break;
+		case 2:
+			return m_divide_remainder & 0xFF;
+			break;
+		case 3:
+			return (m_divide_remainder >> 8) & 0xFF;
+			break;
+		case 4:
+			return m_divide_quotient & 0xFF;
+			break;
+		case 5:
+			return (m_divide_quotient >> 8) & 0xFF;
+			break;
+		default:
+			return 0;
```

Comment:
Please use lowercase hex, like the rest of the file.

Reply:
Also, this will fail to build with “unreachable code” errors for all the `break` statements following `return` statements.

Reply:
Made the changes in a093905, thanks.